server/service/src/sync/translations/report.rs

```diff
@@ -16,7 +20,7 @@ impl SyncTranslation for OmReportTranslator {
         "om_report"
     }
     fn pull_dependencies(&self) -> Vec<&str> {
-        vec![]
+        vec![OmFormSchemaTranslation.table_name()]
```

Comment:
Added this as a dependency to reports. Though I'm not sure if strictly required as a dependency

Reply:
Not necessary since it isn't constraint by FK